build.sbt

```diff
@@ -3967,7 +3967,8 @@ lazy val `std-benchmarks` = (project in file("std-bits/benchmarks"))
       Map(
         runtimeModName -> Seq(
           "ALL-UNNAMED",
-          (`benchmarks-common` / javaModuleName).value
+          (`benchmarks-common` / javaModuleName).value,
+          (`bench-processor` / javaModuleName).value
```

Comment:
This may be handy in #10714 - I also had problems with access restrictions in the PR.